### PR TITLE
Add raycast-urbandictionary-word-of-the-day extension

### PR DESCRIPTION
**Pull request by r0hin**
## Description

The extension adds three new commands
1: inline command which fetches the current word of the day from UrbanDictionary public API.
2: view command which displays the definition of the word of the day, also using the UrbanDictionary public API.
3: view command which displays the first entry of a list of random words, fetching using the UrbanDictionary public API.

## Screencast

image

image

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Would it make sense to update the [current extension](https://www.raycast.com/Sheraff/urban-dictionary) with the new commands - it's made by @Sheraff 🙂

**Comment by Sheraff** [comment]
> Would it make sense to update the [current extension](https://www.raycast.com/Sheraff/urban-dictionary) with the new commands - it's made by @Sheraff 🙂

TL;DR: do as you please!

Sorry I know I'm kind of the owner of the Urban Dictionary extension, but I'm not using raycast anymore and so I don't really have the motivation to work on extensions.

**Comment by r0hin** [comment]
I:

1) think the general purpose of the extension is different (search vs word of the day).
2) wouldn't want to burden Flo with extra work and approvals if he's not motivated.

@pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥
